dawn/CMakeLists.txt

```diff
@@ -123,7 +124,7 @@ endfunction()
 
 # Protobuf
 if(NOT DAWN_BUILD_PROTOBUF)
-  find_package(Protobuf 3.4.0 CONFIG)
```

Comment:
what are the implications of this?

Reply:
It is necessary to remove CONFIG if we want to have protobuf found from system packages. If you want to use a custom protobuf version that's still fine, but pass `Protobuf_DIR` appropriately. It should break nothing. @havogt do you see an issue with this?